source/regression2.md

```diff
@@ -517,7 +495,7 @@ lm_plot_final = (
 )
 
 lm_plot_final += lm_plot_final.transform_regression("sqft", "price").mark_line(
-    color="blue"
+    color="#ff7f0e"
```

Comment:
just before this, we use `+=` in generating the plot. Has this been introduced yet? It might have for assigning values, but I don't think we have for plots yet

Reply:
I'll open an issue to do a larger pass through the book for this later -- punt for now on this chapter.